gempython/utils/db/settings.py

```diff
@@ -0,0 +1,17 @@
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "gempython.utils.db.settings.local")
+SECRET_KEY = 'ef7$_)r0$#(x)ovd^m_6_bevsonkaxalvrhds9@ilb_p0o&-xn'
+
+INSTALLED_APPS = [
+    'gempython.utils.ldqm_db',
+    ]
+DATABASES = {
+    'default': {
+        'ENGINE':  'django.db.backends.mysql',
+        'NAME':     'ldqm_tif_qc8_db',
+        'PORT':     3306,
+        'HOST':     'cosmicstandtif',
+        'USER':     'gemdaq',
+        'PASSWORD': 'gemdaq',
+    }
+}
```

Comment:
Did you `git mv` these files, the history is not present...
Actually, now that I look, it seems it wasn't for when I `git mv`d from `setup/scripts/python` to `gempython` either... hmm, this is not the behaviour that I expect...